reporter/amqp/amqp.go

```diff
@@ -0,0 +1,201 @@
+/*
+Package amqp implements a RabbitMq reporter to send spans to a Rabbit server/cluster.
+*/
+package amqp
+
+import (
+	"encoding/json"
+	"fmt"
+	"log"
+	"os"
+
+	"github.com/streadway/amqp"
+
+	"github.com/openzipkin/zipkin-go/model"
+	"github.com/openzipkin/zipkin-go/reporter"
+)
+
+// defaultRmqRoutingKey/Exchange/Kind sets the standard RabbitMQ queue our Reporter will publish on.
+const (
+	defaultRmqRoutingKey = "zipkin"
+	defaultRmqExchange   = "zipkin"
+	defaultExchangeKind  = "direct"
+)
+
+// rmqReporter implements Reporter by publishing spans to a RabbitMQ exchange
+type rmqReporter struct {
+	e        chan error
+	channel  *amqp.Channel
+	conn     *amqp.Connection
+	Exchange string
```

Comment:
I am curious about `Exchange` and `Queue` being exported.

Reply:
> I am curious about `Exchange` and `Queue` being exported.

Done, I made it unexported